### DISCUSSION TITLE
Remove header and footer and default to /docs/ ?

### DISCUSSION THREAD

**Discussion by cotjoey** [google/docsy#678] [Q&A] [answered]
Hello,

I was wondering if I could disable the header/footer from within the config.yaml so that I end up with just the content/en/docs showing in the browser?

See screenshot for how I'd like the site to render.

Thank you,
Joey

![Screen Shot 2021-09-02 at 12 16 14 AM](https://user-images.githubusercontent.com/11093392/131776107-3e201d2a-e5f8-4945-b342-1c5e9c730077.png)

**Comment by LisaFC** [comment, marked as answer]
So you can create a "just docs" (or mostly docs) site by following the instructions here: https://www.docsy.dev/docs/adding-content/content/#alternative-site-structure (example site: https://github.com/gwatts/mostlydocs/)

Though your site would still have a header/footer, they're part of the default page layout. You could remove them, but you'd need to create your own layout files.
